unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+namespace DCL
+{
+    public class ReleaseParticlesOnFinish : MonoBehaviour
+    {
+        private const int INDEX_AMOUNT = 10;
+
+        [SerializeField] private ParticleSystem particles;
+        private PoolableObject poolable = null;
+        private int index;
+        private bool initialized;
+
+
+        public void Initialize(PoolableObject poolableObject)
+        {
+            if (particles == null)
+            {
+                Debug.LogError($"The script: {nameof(ReleaseParticlesOnFinish)}, " +
+                    $"should never be initialized without particles attached: Aborting initialization");
+                Destroy(this);
+                return;
+            }
+
+            poolable = poolableObject;
+            index = Random.Range(0, INDEX_AMOUNT);
+            initialized = false;
+        }
+
+        public void FixedUpdate()
```

Comment:
why FixedUpdate? what about Update?

Reply:
@lorux0  It was like this before and I found no reason to change it ,as goes through the physics cycle, should be no issue having it here, if you feel a change is due, would be fine for me

Reply:
i think there is not reason to keep in a fixed step updating, it should work through the frame rate loop

Reply:
Will update it then ^^